docs/content/en/docs/reference/clusterspec/optional/irsa.md

```diff
@@ -83,41 +110,65 @@ Set the remaining fields in [cluster spec]({{< relref "../vsphere/" >}}) as requ
 1. The cluster provisioning workflow generates a pair of service account signing keys. Retrieve the public signing key generated and used by the cluster, and create a keys.json document containing the public signing key. 
 
     ```bash
-    kubectl get secret ${CLUSTER_NAME}-sa -n eksa-system -o jsonpath={.data.tls\\.crt} | base64 --decode > ${CLUSTER_NAME}-sa.pub
-    wget https://raw.githubusercontent.com/aws/amazon-eks-pod-identity-webhook/master/hack/self-hosted/main.go -O keygenerator.go
-    go run keygenerator.go -key ${CLUSTER_NAME}-sa.pub | jq '.keys += [.keys[0]] | .keys[1].kid = ""' > keys.json
+    git clone https://github.com/aws/amazon-eks-pod-identity-webhook
+    cd amazon-eks-pod-identity-webhook
+    kubectl get secret ${CLUSTER_NAME}-sa -n eksa-system -o jsonpath={.data.tls\\.crt} | base64 --decode > ${CLUSTER_NAME}-sa.pub    
+    go run ./hack/self-hosted/main.go -key ${CLUSTER_NAME}-sa.pub | jq '.keys += [.keys[0]] | .keys[1].kid = ""' > keys.json
     ```
-1. Upload the keys.json document to the s3 bucket.
+2. Upload the keys.json document to the s3 bucket.
```

Comment:
You can keep this number as "1.".  As long as the indents are right from the previous step (which they appear to be), it will number that second step as "2.". Using "1." on all the steps keeps from having to renumber long procedures when you add a step.

Reply:
Thanks for your observations. I've updated and committed the changes.